lstore/db.py

```diff
@@ -15,6 +16,8 @@ def __init__(self, path="ECS165 DB"):
         self.no_path_set = True
         self.lock_manager = LockManager()
         atexit.register(self.__remove_db_path)
+
+        self.db_lock = threading.Lock()
```

Comment:
there is only one db and transaction worker cannot access db so I don't think we need to lock db